### PR TITLE
COMMERCE-4700 Fixed form values look up in FS Product detail when a p…

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
…roduct has options

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/form_handlers/metal_js.jspf [hunk 1]
```diff
@@ -61,24 +61,37 @@
 		})
 	}
 
+	function getFormValues(form) {
+		return !form ? [] : form.getImmediateFields().map(
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/form_handlers/metal_js.jspf:65]: @gianmarcobrunialti I know this is correct but is hard to read, a negational condiction in a Ternary operation, is a bit complex to read

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**